terraform/grafana.tf

```diff
@@ -0,0 +1,116 @@
+resource "aws_cloudwatch_log_group" "task" {
+  name = "platform/grafana"
+}
+
+data "template_file" "definition" {
+  template = "${file("grafana.json.template")}"
+
+  vars {
+    log_group_region = "${var.aws_region}"
+    log_group_name   = "${aws_cloudwatch_log_group.task.name}"
+  }
+}
+
+resource "aws_ecs_task_definition" "task" {
+  family                = "grafana_task_definition"
+  container_definitions = "${data.template_file.definition.rendered}"
+  task_role_arn         = "${module.ecs_grafana_iam.task_role_arn}"
+
+  volume {
+    name = "grafana"
+    host_path = "/ecs/grafana"
```

Comment:
If Grafana cannot function without a local filesystem then we should talk further about the options.